packages/atoms/src/Button/IconButton.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { Icon, IconName } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { ButtonColorScheme, ButtonSize } from './Button';
+
+type Props = {
+    iconName: IconName;
+    onPress: () => void;
+    isRounded?: boolean;
+    colorScheme?: ButtonColorScheme;
+    size?: ButtonSize;
+};
+
+type StyleProps = {
+    isRounded: boolean;
+    size: ButtonSize;
+    colorScheme: ButtonColorScheme;
+};
+
+const iconButtonStyle = prepareNativeStyle<StyleProps>(
+    (utils, { isRounded, colorScheme, size }) => {
+        const colorSchemeStyles = {
+            primary: {
+                backgroundColor: utils.colors.green,
+                color: utils.colors.white,
+            },
+            gray: {
+                backgroundColor: utils.colors.gray300,
+                color: utils.colors.gray700,
+            },
+        };
+
+        const sizeStyles = {
+            small: {
+                width: 16,
+                height: 16,
+                borderRadius: isRounded ? utils.borders.radii.round : utils.borders.radii.small / 2,
+            },
+            medium: {
+                width: 32,
+                height: 32,
+                borderRadius: isRounded ? utils.borders.radii.round : utils.borders.radii.small,
+            },
+            large: {
+                width: 48,
+                height: 48,
+                borderRadius: isRounded ? utils.borders.radii.round : utils.borders.radii.small,
+            },
+        };
+        return {
+            justifyContent: 'center',
+            alignItems: 'center',
+            ...colorSchemeStyles[colorScheme],
+            ...sizeStyles[size],
+        };
+    },
+);
+
+export const IconButton = ({
+    iconName,
+    onPress,
+    colorScheme = 'primary',
+    size = 'medium',
+    isRounded = false,
+}: Props) => {
```

Comment:
```suggestion
}: IconButtonProps) => {
```
We use the naming ComponentNameProps for the component type. At least it is everywhere in atoms so keep it, please.

Reply:
I would argue that since it's not exported, it should be just props because it's obvious that it's props to the icon button component hence shouldn't be needed to specify. I will rename it I'm outnumbered and @Nodonisko thinks we should adhere to this naming convention but it doesn't make a difference as to what it's refering to so I wouldn't worry about it too much I guess.